.github/workflows/main.yml

```diff
@@ -72,10 +72,19 @@ jobs:
         run: ./02_build.sh
 
       - name: 3. export ...
+        id: export
         working-directory: knownprojects_build
-        env:
-          SENDER: ${{ github.event.sender.login }}
-          NOTES: ${{ github.event.inputs.run_notes }}
-        run: | 
-          echo $NOTES > run_notes.txt
-          ./03_export.sh $SENDER
+        run: ./03_export.sh ${{ github.event.sender.login }}
+      
+      - name: 4. comment on PR
```

Comment:
This is commenting on the db-knownprojects repo, rather than the db-knownprojects-data repo.

For example, the output in the [db-knownprojects-data #46](https://github.com/NYCPlanning/db-knownprojects-data/pull/46) has no comment, and the comment ended up in the db-knownprojects repo #46, which is unrelated.

Reply:
ohhhh, we need to specify the repo then